src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/ReportedAdjudicationIntTest.kt

```diff
@@ -101,6 +101,46 @@ class ReportedAdjudicationIntTest : IntegrationTestBase() {
       .jsonPath("$.content[1].incidentDetails.locationId").isEqualTo(721850)
   }
 
+  @Test
+  fun `return a page of reported adjudications completed in the current agency`() {
+    val intTestData = IntTestData(webTestClient, jwtAuthHelper, bankHolidayApiMockServer, prisonApiMockServer)
```

Comment:
We could probably abstract this into the TestBase class, but that's a future change